modules/va_forms/app/workers/va_forms/form_reloader.rb

```diff
@@ -6,82 +6,69 @@ module VaForms
   class FormReloader
     include Sidekiq::Worker
 
-    BASE_URL = 'https://www.va.gov'
-
-    def initialize
-      @processed_forms = []
-    end
+    FORM_BASE_URL = 'https://www.va.gov'
 
     def perform
-      load_page(current_page: 0)
-      mark_stale_forms
-    end
-
-    def mark_stale_forms
-      processed_form_names = @processed_forms.map { |f| f['form_name'] }
-      missing_forms = VaForms::Form.where.not(form_name: processed_form_names)
-      missing_forms.find_each do |form|
-        form.update(valid_pdf: false)
+      query = File.read(Rails.root.join('modules', 'va_forms', 'config', 'graphql_query.txt'))
+      body = { query: query }
+      response = connection.post('graphql', body.to_json)
+      forms_data = JSON.parse(response.body)
+      processed_forms = []
+      forms_data.dig('data', 'nodeQuery', 'entities').each do |form|
+        va_form = build_and_save_form(form)
+        processed_forms << va_form
+      rescue
+        next
```

Comment:
do we wanna do any kinda logging here? i'm worried that they may send us something formatted in a way we don't expect, and build_and_save throw starts throwing exceptions that are silently ignored. if build_and_save failed for all of them, we'd notice, but if it was just a 1/4 or a 1/3 we might not (exs: uri normalization fails, date parsing fails)

Reply:
we could log to Sentry maybe. Since its a background job no one is actively looking at it. The goal of next was to not allow one bad form the break the loop

Reply:
>The goal of next was to not allow one bad form the break the loop
for sure. I'd be down for just sending the exception to sentry, and, if we see a bunch, take a peek and see if we need to change something up b/c the data coming in is goofier than we thought

Reply:
added